services/harmony/app/frontends/service-image-tags.ts

```diff
@@ -121,6 +122,44 @@ async function validateTaggedImageIsReachable(
   return isReachable;
 }
 
+/**
+ * Returns value of the enabled field of service_deployment table.
+ * @param tx - The database transaction
+ * @returns The boolean value of the enabled field
+ */
+async function getEnabled(): Promise<boolean> {
+  let enabled = true;
+  let results = null;
+  await db.transaction(async (tx) => {
+    results = await tx('service_deployment').select('enabled');
+  });
+
+  if (results[0].enabled === 0 || results[0].enabled === false) {
+    enabled = false;
+  }
+  return enabled;
+}
+
+/**
+ * Validate that the service deployment is enabled
+ * @param res - The response object - will be used to send an error if the validation fails
+ * @param url - The URL of the image including tag
+ * @returns A Promise containing `true` if the tagged image is reachable, `false` otherwise
+ */
+async function validateServiceDeploymentIsEnabled(
+  req: HarmonyRequest,
+  res: Response,
+): Promise<boolean> {
+  const enabled = await getEnabled();
+  if (!enabled) {
+    res.statusCode = 403;
```

Comment:
We should use 503 here - this is a server side not allowed, not a user forbidden issue.